index.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-use-before-define */
+/* eslint-disable import/no-cycle */
```

Comment:
- [ ] As the previous code reviewer mentioned, please fix the linters and remove the `eslint-disable` lines from the Javascript files.